stable/kong/values.yaml

```diff
@@ -51,10 +51,19 @@ proxy:
   annotations: {}
   #  service.beta.kubernetes.io/aws-load-balancer-proxy-protocol: "*"
 
-  # HTTPS traffic on the proxy port
-  useTLS: true
-  servicePort: 8443
-  containerPort: 8443
+  # HTTP plain-text traffic
+  http:
+    enabled: true
+    servicePort: 80
+    containerPort: 8000
+    nodePort: 32080
```

Comment:
@sekka1 Seems like CI failing due to this. Let's get rid off the NodePort for both http and tls. k8s will auto-assign.

Reply:
Ok...I guess that is why right below it, the nodePort is commented out?  

Removed the older nodePort comment and then I commented this one out.  Is that sufficient or should I remove it completely?